<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>413</width>
    <height>547</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="Body">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>391</width>
      <height>471</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="body_tab">
     <attribute name="title">
      <string>Body</string>
     </attribute>
     <widget class="QComboBox" name="comboBox">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>30</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="acceptDrops">
       <bool>true</bool>
      </property>
      <property name="editable">
       <bool>true</bool>
      </property>
      <item>
       <property name="text">
        <string>Biped</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Quadruped</string>
       </property>
      </item>
     </widget>
     <widget class="QCheckBox" name="body_sym_check">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>80</y>
        <width>70</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Symmetric</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="create_temp_points">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>70</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Create Temorary Points</string>
      </property>
      <property name="autoDefault">
       <bool>true</bool>
      </property>
      <property name="+command" stdset="0">
       <string>&quot;createSpheres()&quot;</string>
      </property>
     </widget>
     <widget class="Line" name="line_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>110</y>
        <width>401</width>
        <height>20</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QCheckBox" name="spine_ik_check">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>150</y>
        <width>41</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>IK</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="spine_ik_with_fk_ckeck">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>150</y>
        <width>111</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Ik with FK Controls</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="spine__fk_ckeck">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>150</y>
        <width>70</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>FK</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="spine_stretch_ckeck">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>150</y>
        <width>70</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Stretch</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="arm_ik_check">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>210</y>
        <width>41</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>IK</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="arm_fk_check">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>210</y>
        <width>41</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>FK</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="arm_stretch_check">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>210</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Stretch</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="arm_bendy_check">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>210</y>
        <width>70</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Bendy</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="finger_number_val">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>260</y>
        <width>42</width>
        <height>22</height>
       </rect>
      </property>
      <property name="value">
       <number>4</number>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>260</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Number of Fingers</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="finger_cup_check">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>260</y>
        <width>70</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Cup</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="finger_sticky_check">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>260</y>
        <width>70</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Sticky</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="leg_ik_check">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>320</y>
        <width>41</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>IK</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="leg_fk_check">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>320</y>
        <width>41</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>FK</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="leg_stretch_check">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>320</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Stretch</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="leg_bendy_check">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>320</y>
        <width>70</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Bendy</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>120</y>
        <width>47</width>
        <height>31</height>
       </rect>
      </property>
      <property name="lineWidth">
       <number>2</number>
      </property>
      <property name="text">
       <string>SPINE</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>180</y>
        <width>47</width>
        <height>31</height>
       </rect>
      </property>
      <property name="lineWidth">
       <number>2</number>
      </property>
      <property name="text">
       <string>Arms</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>230</y>
        <width>47</width>
        <height>31</height>
       </rect>
      </property>
      <property name="lineWidth">
       <number>2</number>
      </property>
      <property name="text">
       <string>Hand</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>290</y>
        <width>47</width>
        <height>31</height>
       </rect>
      </property>
      <property name="lineWidth">
       <number>2</number>
      </property>
      <property name="text">
       <string>Legs</string>
      </property>
     </widget>
     <widget class="Line" name="line_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>280</y>
        <width>381</width>
        <height>20</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="Line" name="line_4">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>170</y>
        <width>391</width>
        <height>20</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>20</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Body Type</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>340</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="lineWidth">
       <number>2</number>
      </property>
      <property name="text">
       <string>Foot </string>
      </property>
     </widget>
     <widget class="QCheckBox" name="leg_ik_check_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>370</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>FootRoll</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="leg_ik_check_4">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>370</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Tilt</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="leg_ik_check_5">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>370</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Rock</string>
      </property>
     </widget>
     <widget class="QFrame" name="frame">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>111</width>
        <height>101</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
     <widget class="QPushButton" name="create_temp_points_3">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>400</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Create Controller</string>
      </property>
      <property name="autoDefault">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="create_temp_points_4">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>400</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Create Rig</string>
      </property>
      <property name="autoDefault">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="face_tab">
     <attribute name="title">
      <string>Face</string>
     </attribute>
     <widget class="QPushButton" name="create_temp_points_5">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>240</y>
        <width>171</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Create Joints, Locs and Curves</string>
      </property>
      <property name="autoDefault">
       <bool>true</bool>
      </property>
      <property name="+command" stdset="0">
       <string>&quot;runJointToVertex()&quot;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>361</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>1. Create locator, name it center and move it to the eyes center</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="prefixOne_lineEdit">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>180</y>
        <width>271</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>361</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>2. Select the vertices and add them to the appropriate field</string>
      </property>
     </widget>
     <widget class="QPushButton" name="create_temp_points_6">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>320</y>
        <width>131</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Create Connections</string>
      </property>
      <property name="autoDefault">
       <bool>true</bool>
      </property>
      <property name="+command" stdset="0">
       <string>&quot;runWireDeformerScript()&quot;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>180</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>3.  Prefix 1:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>290</y>
        <width>341</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>4. Deform the cubic Curve to match the linear one</string>
      </property>
     </widget>
     <widget class="QPushButton" name="upperVertSel_add_button">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>100</y>
        <width>121</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Save UpperVertices</string>
      </property>
      <property name="autoDefault">
       <bool>true</bool>
      </property>
      <property name="+command" stdset="0">
       <string>&quot;getUpperVertsList()&quot;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="upperVertSel_add_button_2">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>100</y>
        <width>121</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Save LowerVertices</string>
      </property>
      <property name="autoDefault">
       <bool>true</bool>
      </property>
      <property name="+command" stdset="0">
       <string>&quot;getLowerVertsList()&quot;</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="prefixTwo_lineEdit">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>210</y>
        <width>271</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>210</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string> Prefix 2:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="controlAmount_lineEdit">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>140</y>
        <width>41</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>5</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_15">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>140</y>
        <width>121</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>3. Number of controllers:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="upperVertSel_add_button_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>40</y>
        <width>121</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Create Locator</string>
      </property>
      <property name="autoDefault">
       <bool>true</bool>
      </property>
      <property name="+command" stdset="0">
       <string>&quot;createMiddleLoc()&quot;</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="spine_tab">
     <attribute name="title">
      <string>Spine</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="arm_tab">
     <attribute name="title">
      <string>Arms</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="leg_tab">
     <attribute name="title">
      <string>Legs</string>
     </attribute>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
